### DISCUSSION TITLE
:lab Support for Gitmoji 😄

### DISCUSSION THREAD

**Discussion by albin-sunnanbo-sveasolar** [gitextensions/gitextensions#11616] [Ideas]
### Feature description

It would be really nice to have gitmoji-support.
Both displaying in the git log and autocomplete/search when typing commit messages (similar to https://chromewebstore.google.com/detail/gitmoji-browser-extension/lkjogeoldakjceempbkdahkojohmbaja?pli=1)

### Environment

- Git Extensions 4.2.1.17611
- Build b0c0b284876b63c52c77195ada82a85bfadbb083
- Git 2.42.0.windows.1
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.27
- DPI 144dpi (150% scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.27 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.16 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 8.0.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by pmiossec** [comment]
Can you describe the feature in a little more details?

How do you expect to see it working? I mean, the commit message that will be stored will have the shortcode ( e.g. ":bug:") or it should be replaced by the unicode char (e.g. "🐛")?

**Comment by albin-sunnanbo-sveasolar** [comment]
Github and other client supporting gitmoji store the plain text ":bug:" version

**Comment by CamiloBernal** [comment]
I agree, it would be great to have support for gitmoji: https://gitmoji.dev/

**Reply by CamiloBernal** [reply]
And I would complement it with this: https://github.com/gitextensions/gitextensions/discussions/11662
